### PR TITLE
PARQUET-740: Introduce editorconfig

### PR DESCRIPTION
**Pull request by nielsbasjes**
I examined some of the code and as far as I can tell this specification matches what has been done in this project so far. Please verify.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+indent_size = 2
+
+[*.yaml]
+indent_size = 2
+
+[*.xml]
+indent_size = 2
+
+# Tab indentation (no size specified)
+[Makefile]
```
Comment (julienledem) [inline .editorconfig]: Makefile for?

**Comment by julienledem** [comment]
+1

File: .editorconfig (hunk 1 above)
Comment (nielsbasjes) [inline .editorconfig]: Because I copied this file from an other project I wrote.
Remove it? Or leave it as it is now?

Comment (julienledem) [inline .editorconfig]: that's fine

**Pull request closed without merging**